static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -41,9 +41,19 @@
         // Removes extra spacing
         display: block;
 
+        // Aspect ratio: 16:3
+        @include mq($until: phablet) {
+            width: 50vw;
+            // height: auto doesn't work in Safari
+            height: calc(3/16 * 46vw);
```

Comment:
any link to webkit bugzilla? so we could keep track of the issue.
why `calc(3/16 * 46vw)` and not `calc(3/16 * 50vw)`?